src/back/routes/vestings.ts

```diff
@@ -0,0 +1,38 @@
+import handleAPI from 'decentraland-gatsby/dist/entities/Route/handle'
+import routes from 'decentraland-gatsby/dist/entities/Route/routes'
+import { Request } from 'express'
+
+import { VestingInfo, getVestingContractData } from '../../clients/VestingData'
+import { validateAddress } from '../utils/validations'
+
+export default routes((router) => {
+  router.post('/vesting', handleAPI(getVestingInfo))
+})
+
+const compareVestingInfo = (a: VestingInfo, b: VestingInfo): number => {
```

Comment:
Maybe we should have a VestingService that handles using the client and sorting and whatever needs to be done

Reply:
Agree. I'm in for having a _cleaner_ routes file.